src/components/StyleGuide/Typography.mdx

```diff
@@ -47,31 +55,297 @@ set the specific `<h>` tag that you require.
   mb="m"
   p="s"
 >
-  <Heading level="one">Heading Level 1</Heading>
+  <Heading level="h1">Heading Level h1</Heading>
+  <Heading level="h2" border="none">
+    Heading Level h2
+  </Heading>
+  <Heading level="h3">Heading Level h3</Heading>
+  <Heading level="h4">Heading Level h4</Heading>
+  <Heading level="h5">Heading Level h5</Heading>
+  <Heading level="h6">Heading Level h6</Heading>
+</Box>
+}
+
+#### Deprecated Options
+
+The older `level` values and the associated styles are still available, but the
+values should be consdidered deprecated and they should only used to fulfill
+older design requirements.
+
+{
+
+<Box
+  border="1px"
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="s"
+>
+  <Heading level="one">Heading Level one (deprecated)</Heading>
   <Heading level="two" border="none">
-    Heading Level 2
+    Heading Level two (deprecated)
+  </Heading>
+  <Heading level="three">Heading Level three (deprecated)</Heading>
+  <Heading level="four">Heading Level four (deprecated)</Heading>
+  <Heading level="five">Heading Level five (deprecated)</Heading>
+  <Heading level="six">Heading Level six (deprecated)</Heading>
+</Box>
+}
+
+### Size Options
+
+Each native heading level has a default style and size, but those styles can be
+overridden using the `size` prop.
+
+The size options are not tied to specific native heading levels and may be used
+to override the default size and styling of any native HTML heading elements.
+For example, the `h1` element has the `h1` style applied by default, but the
+`size` prop can be used to apply a smaller font size style.
+
+#### Display 1
+
+{
+
+<List type="ul">
+  <li>
+    This style SHOULD be used for display headings meant to give an immediate
+    great impact.
+  </li>
+  <li>
+    This style SHOULD be used with short headlines. If headline text is more
+    than five words, consider using a smaller heading style.
+  </li>
+</List>
+}
+
+|                    | JS Token                   | CSS Token                                        | Value            |
+| ------------------ | :------------------------- | :----------------------------------------------- | :--------------- |
+| **Font Size**      | `desktop.heading.display1` | `var(--nypl-fontSizes-desktop-heading-display1)` | `4.25rem` (68px) |
+| **Font Weight**    | `heading.display1`         | `var(--nypl-fontWeights-heading-display1)`       | `200` (thin)     |
```

Comment:
This now feels like we are not being consistent with the variable chaining values. Why does font weight not have `desktop` as well? (rhetorical)